# header
uint32 counter
time timestamp

# x, y of the most likely current location of the robot in metres, relative to
# the top left of the global map
float64 x
float64 y
float64 rotation

